content/en/docs/tasks/configure-pod-container/pull-image-private-registry.md

```diff
@@ -206,6 +206,7 @@ kubectl get pod private-reg
 
 * Learn more about [Secrets](/docs/concepts/configuration/secret/).
 * Learn more about [using a private registry](/docs/concepts/containers/images/#using-a-private-registry).
+* Learn more about [adding image pull secrets to a service account](/docs/tasks/configure-pod-container/configure-service-account/#add-imagepullsecrets-to-a-service-account).
```

Comment:
```suggestion
* Learn about [adding image pull secrets to a service account](/docs/tasks/configure-pod-container/configure-service-account/#add-imagepullsecrets-to-a-service-account).
```

this is also fine to merge as-is.

Reply:
Thanks for the PR @rajeshdeshpande02 .

/approve